src/net/sourceforge/kolmafia/AdventureResult.java

```diff
@@ -578,6 +592,14 @@ public static final AdventureResult parseResult(final String s) {
         statname = parsedGain.nextToken();
       }
 
+      if (statname.startsWith("energy") || statname.startsWith("Energy")) {
+        return new AdventureResult(AdventureResult.ENERGY, (int) modifier);
```

Comment:
Non-blocking comment.  This could be one clause - statname.toLowerCase().startsWith("energy")

Similar comment for scrap.

Reply:
OK.